lib/base_service.ts

```diff
@@ -34,6 +35,7 @@ export interface UserOptions {
   username?: string;
   password?: string;
   api_key?: string;
+  platform_api_key?: string;
```

Comment:
I suspect you're already working on this change, but I think we settled on something like `iam_apikey` instead.

Reply:
Yep! Already made this change 👍